[Doc] correct ceph adoption prerequisites (#745)
openstack-k8s-operators/data-plane-adoption: docs_user/modules/proc_creating-a-ceph-nfs-cluster.adoc

You can deploy the new clustered NFS service on your existing CephStorage nodes or HCI nodes, or on new hardware that you enrolled in the {Ceph} cluster.

. If you deployed your {Ceph} nodes with {OpenStackPreviousInstaller}, propagate the `StorageNFS` network to the target nodes where the `ceph-nfs` service is deployed.

> .Note: When target nodes are not managed by Director, it's not possible to use this procedure/tool to configure the network. All the required networks must be manually configured by the administrator.

Reviewer: same as before.
@klgill to review/suggest the right way of passing this note.

> .Note: When target nodes are not managed by Director, it's not possible to use this procedure/tool to configure the network. All the required networks must be manually configured by the administrator.

Reviewer 2: ```suggestion
+
[NOTE]
If the target nodes are not managed by director, you cannot use this procedure to configure the network. An administrator must manually configure all the required networks.
```

Reviewer: @katarimanojk FYI ^
